data/shobhitexe.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Shobhit gupts",
+  "type": "personal",
+  "bio": "Dev | Passionate about tech, Web3 & gaming | Exploring the future",
+  "tags": [
+    "Open Source",
+    "Solidity",
+    "React.js",
+    "TypeScript",
+    "MongoDB",
+    "Express"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/shobhitexe" },
+    { "icon": "FaGithub", "url": "https://github.com/shobhitexe" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/shobhitx/" },
+    {
+      "icon": "FaDollarSign",
+      "url": "https://github.com/sponsors/Pradumnasaraf"
```

Comment:
Overall, the code looks good, but I noticed that the link to the Github Sponsor is incorrect. 

Please update the link to the correct Github Sponsor page.

Wrong Link:  ` https://github.com/sponsors/Pradumnasaraf`

Correct Link Format: ` https://github.com/sponsors/[username] `

Reply:
Yes it supposed to be not there resolved